packages/shared/lib/clients/sync.client.ts

```diff
@@ -53,6 +53,10 @@ class SyncClient {
     }
 
     private static async create(): Promise<SyncClient | null> {
+        if (isTest()) {
```

Comment:
What were you seeing here that necessitated this?

Reply:
Temporal is not spawn in CI, if we don't pass params it won't try to connect. Let me know if you have a better solution